dotcom-rendering/src/components/AdSlot.tsx

```diff
@@ -227,24 +225,21 @@ const mobileFrontAdStyles = css`
 	}
 `;
 
-const frontsBannerAdContainerStyles = css`
+const frontsBannerPaddingHeight = 20;
+
+const frontsBannerAdTopContainerStyles = css`
 	display: flex;
 	justify-content: center;
-	background-color: ${neutral[97]};
-	min-height: ${250 + labelHeight}px;
+	min-height: ${250 + labelHeight + frontsBannerPaddingHeight}px;
```

Comment:
While we're here is it worth extracting 250 to a constant e.g. `mpuHeight`? We use it in quite a few places in `AdSlot.tsx`

Reply:
Yeah I like this idea